docs/self-managed/setup/deploy/other/docker.md

```diff
@@ -218,3 +193,7 @@ You can also add a Connector JAR using volumes:
 ```shell
 docker run --rm --name=connectors -d -v $PWD/connector.jar:/opt/app/ camunda/connectors:x.y.z
 ```
+
+### Web Modeler
+
+You can also find more information on the supported [configuration variables](/self-managed/modeler/web-modeler/configuration/configuration.md).
```

Comment:
I know I previously suggested this but reading through the page now this section seems unnecessary - let's just completely remove it.